server/static/js/admin/controllers.js

```diff
@@ -298,14 +298,25 @@ app.controller("CourseDetailCtrl", ["$scope", "$stateParams", "Course",
   }
   ]);
 
-app.controller("CourseNewCtrl", ["$scope", "Course",
-  function ($scope, Course) {
+app.controller("CourseNewCtrl", ["$scope", "$state", "$window", "Course",
+  function ($scope, $state, $window, Course) {
     $scope.course = {};
     $scope.test = {'test':3};
```

Comment:
The two params? We need $state for transitionTo and $window for the alert, though.